links/libraryofcongress.js

```diff
@@ -8,6 +8,7 @@ module.exports = {
   authoritative: true,
 
   citations: {
+    /*
```

Comment:
What's the disposition of this function?

Reply:
The function remains a somewhat helpful example of how we might generate links to a Library of Congress resource for the Statutes at Large, but because it links to a main landing page and not a particular volume or page of the SaL, I commented it out because it's not very helpful.